.github/workflows/purefinance_deployment.yaml

```diff
@@ -27,13 +27,18 @@ jobs:
           cd site
           BASE_PATH="/pure.finance" npm run build
 
+      - name: Content Files
+        run: |
+          mkdir -p pure.finance
+          rsync -av  purefinance/site/out/ pure.finance/
+
```

Comment:
@dhidalgX added this because the whole path to the source files was getting uploaded.
I found there might be no need for this step. The action appleboy/scp-action has an option called `strip_components` that should help us getting rid of the whole dir structure that get's uploaded.

Reply:
@leanazulyoro Thanks Lean, let me test it.

Reply:
Test completed strip_components: 3